### PR TITLE
Rename warmer function

### PR DESCRIPTION
**Pull request by juanjoDiaz**
This is a follow up to my previous PR.

It renames the warmer function to `service-stage-warmup-plugin` instead of `warmup-plugin-service-stage` so it's consistent with serverless naming convention.

I have several serverless projects running on the same AWS account/region. With this new name I can sort the lambdas alphabetically and see what belongs to which project. Before, all my projects where ordered but the warmer lambdas for all the projects where just together at the bottom.

As usual, open to suggestion and changes. :)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -59,49 +97,41 @@ class WarmUP {
     /** Default options */
     this.warmup = {
       memorySize: 128,
-      name: 'warmup-plugin-' + this.serverless.service.service + '-' + this.options.stage,
+      name: this.serverless.service.service + '-' + this.options.stage + 'warmup-plugin',
```
Comment (juanjoDiaz) [inline src/index.js]: This line is the only actual change. Everything else comes from the other pull request. My bad, I should have done it in separate branches from the beginning...

**Comment by goncaloneves** [comment]
@juanjoDiaz thanks for this great PR. 👍 Love the pre-warm, super cool idea!

One question, why the change to compute the static payload with `JSON.stringify()`?
I see that useful if we would want to customise that object. What was your reasoning?

**Comment by juanjoDiaz** [comment]
The reason to use `JSON.stringify()` is to make it very clear that it's a text and not a JSON object. It also makes easier if we want to modify that object in the future.

I got bitten by that one when creating the prewarm function call since I didn't noticed that it was a text. The double backslashes before the double quotes in `"{\\"source\\": \\"serverless-plugin-warmup\\"}"` were not obvious enough for me :)

Other option that could be a good idea is to replace the whole function stuff by a string template so you don't need the `\n' +` at the end of each line. I can also do a PR with that change if you want.

**Comment by goncaloneves** [comment]
Yes using string template will be a good improvement. if you can do it, go for it 👌  After this I will merge and do a major release due to the function name change.

Non-related, do you have a way to send you over a DM?

**Comment by juanjoDiaz** [comment]
Added the string template.

I don't think GitHub support DMs anymore but you can see my email in the author field of any of my commits :)

**Pull request merged**